### PR TITLE
`clean-repo-sidebar` fix: Section disappears if URL includes “releases”

### PR DESCRIPTION
**Pull request by VergeDX**
From #4424

### CODE REVIEW

**Comment by yakov116** [comment]
@VergeDX Thanks for the PR.

Can you try not to include issue numbers in commits as creates unnecessary noise

image

File: source/features/clean-repo-sidebar.tsx [hunk 1]
```diff
@@ -20,7 +20,10 @@ async function cleanLicenseText(): Promise<void> {
 }
 
 async function cleanReleases(): Promise<void> {
-	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]', {waitForChildren: false});
+	// Intent: select "Releases" section in right sidebar.
+	// But if "repository details" - "Website" has a link end with "/releases", it will be wrongly selected.
+	// @see refined-github/issues/4424 for more details.
+	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]:not([role])', {waitForChildren: false});
```
Comment (yakov116) [inline source/features/clean-repo-sidebar.tsx:26]:
```suggestion
	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]:not([role])', {waitForChildren: false}); // :not('[role]') excludes website links that end with "/releases" #4424
```

I did not test the code, however I think the above comment is enough.

Comment (kidonng) [inline source/features/clean-repo-sidebar.tsx:26]: Perhaps

```suggestion
	const sidebarReleases = await elementReady('.BorderGrid-cell h2 a[href$="/releases"]', {waitForChildren: false});
```

Comment (fregante) [reply source/features/clean-repo-sidebar.tsx:26]: Yeah this would work best  since we’re specifically looking for a header link

**Comment by VergeDX** [comment]
Changes performed, no need any comment...?

**Comment by fregante** [comment]
謝謝～

**Pull request merged**